src/docs/pages/ComponentsModal.js

```diff
@@ -0,0 +1,389 @@
+              Because a modal can contain <b>anything you want</b>, you can very
+              simply use it to build an image gallery for example.
+            </p>
+          </Content>
+          <Content>
+            <Button
+              onClick={() => setIsImageModalActive(!isImageModalActive)}
+              isLarge={true}
+              isPrimary={true}
+            >
+              Launch Image Modal
+            </Button>
+            <Modal isActive={isImageModalActive}>
+              <Modal.Background
+                onClick={() => setIsImageModalActive(!isImageModalActive)}
+              ></Modal.Background>
+              <Modal.Close
+                onClick={() => setIsImageModalActive(!isImageModalActive)}
+                className="is-large"
+              ></Modal.Close>
+              <Modal.Content>
+                <img src="../assets/trunx-logotype-white.png" />
+              </Modal.Content>
+            </Modal>
+            <Code language="jsx">
+              {indent`
+            <Modal isActive={isImageModalActive}>
+              <Modal.Background
+                onClick={() => setIsImageModalActive(!isImageModalActive)}
+              ></Modal.Background>
+              <Modal.Close
+                onClick={() => setIsImageModalActive(!isImageModalActive)}
+                className="is-large"
+              ></Modal.Close>
+              <Modal.Content>
+                <img src="../assets/trunx-logotype-white.png" />
+              </Modal.Content>
+            </Modal>
+              `}
+            </Code>
+          </Content>
+          <Title>Modal Card</Title>
+          <Content>
+            <p>
+              If you want a more classic modal, with a <strong>head</strong>, a{" "}
+              <strong>body</strong> and a <strong>foot</strong>, use the{" "}
+              <code>ModalCard</code>.
+            </p>
+          </Content>
+          <Content>
+            <Button
+              isLarge
+              isPrimary
+              onClick={() => setIsCardModalActive(!isCardModalActive)}
+            >
+              Launch example modal
+            </Button>
+
+            <Modal isActive={isCardModalActive}>
+              <Modal.Background
+                onClick={() => setIsCardModalActive(!isCardModalActive)}
+              />
+
+              <Modal.Card>
+                <Modal.Card.Head>
+                  <Level>
+                    <Level.Left>
+                      <Level.Item>
+                        <Modal.Card.Title>Modal Title</Modal.Card.Title>
+                      </Level.Item>
+                    </Level.Left>
+                    <Level.Right>
+                      <Level.Item>
+                        <Delete
+                          onClick={() =>
+                            setIsCardModalActive(!isCardModalActive)
+                          }
+                        />
+                      </Level.Item>
+                    </Level.Right>
```

Comment:
Level.Right wasn't behaving the way I thought it would, by pulling the close button to the right. It's a least level but might be a bug in the level code? Or did something wrong here.

Reply:
in the original bulma docs the `level` class is not used, the markup is something like

```html
<header class="modal-card-head">
  <p class="modal-card-title">Modal title</p>
  <button class="delete" aria-label="close"></button>
</header>
```

however if I use the corresponding trunx components there is still something wrong

<img width="644" alt="Screenshot 2020-10-03 at 12 49 18" src="https://user-images.githubusercontent.com/176730/94989682-e1f2d600-0576-11eb-91a4-f0faa93ffcfc.png">

It looks like the margin bottom is fixed somehow by the Bulma documents CSS. I will try to fix it with a custom class.